CONTRIBUTING.md

```diff
@@ -148,7 +148,7 @@ If you've exhausted all options on trying to fix a failing check, feel free to l
 After the checks in your pull request pass, someone will review your code.
 There may be some discussion and, in most cases, a few iterations will be required to find a solution that works best.
 
-## Afterwards
+## Afterward
```

Comment:
Hmmm, is this right? I tend to see usage of "afterwards" commonly, but not "afterward".